x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_header.scss

```diff
@@ -0,0 +1,26 @@
+.appSearchResultHeader {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  &Column {
+    display: flex;
+    flex-wrap: wrap;
+
+    &--left {
+      justify-content: flex-start;
+
+      & > div {
+        margin-right: 27px;
```

Comment:
EUI var suggestion:

```suggestion
        margin-right: $euiSizeL;
```

Reply:
Gosh, thanks. I did a full sweep of px values but I must have added some styles afterward that I missed. I'll do another sweep.